thrift/src/main/thrift/atoms/interactive.thrift

```diff
@@ -0,0 +1,13 @@
+namespace * contentatom.cta
+namespace java com.gu.contentatom.thrift.atom.interactive
+#@namespace scala com.gu.contentatom.thrift.atom.interactive
+
+struct InteractiveAtom {
+  /* the unique ID will be stored in the `atom` data*/
+  2: required string type
```

Comment:
I think this is a case of a bad thing being copied twice. I think you're safe to start the numbering at 1.